docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition.md

```diff
@@ -631,6 +631,8 @@ Performing surgery by cutting model inputs and outputs from a model is no longer
 Below are examples of model cutting of TensorFlow protobuf, TensorFlow SavedModel, and ONNX formats with the legacy conversion API, compared to achieving the same cut with tools provided by the Tensorflow and ONNX frameworks.
 For PyTorch, TensorFlow 2 Keras, and PaddlePaddle, we recommend changing the original model code to perform the model cut.
 
+Note: This guide does not cover the cutting by input port of an operation. This type of cut is available only in legacy API. 
```

Comment:
Honestly, I don't know what that means. Do we expect most readers to know what we're talking about in this note? If not, then it's mostly distracting IMO.

Reply:
MO provides capability to cut a model by input port of an operation, for example, `--input 1:name_op` or `--output 0:name_op` but this guide does not tell how to handle such case. So we should explicitly tell an user that this case is not covered.
If the current sentence is unclear, let us clarify this case a bit more.

```
This guide does not cover the cutting a model by input port of an operation that MO tool provides using `input` and `output` options, for example, `--input 1:name_op`. 
```

Reply:
Corrected note according to Roman suggestion.

Reply:
OK, that's more clear. Thank you. Ship it!